portfolio/src/main/webapp/maps.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/maps.css">
+    <script src="script.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC5C9lfHXZK0KnxRhO462vInbE-hoGCUrQ"></script>
+  </head>
+  <body onload="createMap();">
```

Comment:
It's generally preferred to not call scripts from html (except from forms). Can we use the DOMContentLoaded event handler instead?
https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event

Reply:
sounds good